app/models/build.js

```diff
@@ -129,7 +129,7 @@ Build.reopen({
   },
 
   restart() {
-    return this.get('ajax').post("/builds/" + (this.get('id')) + "/restart");
+    return Ember.$.post(`/builds/${this.get('id')}/restart`);
```

Comment:
I missed this, it should stay `ajax`, because otherwise will fail. This is an interesting issue, though, the new test didn't catch it, because in Ember CLI Mirage we don't check any headers (like: api version, authentication etc)